### PR TITLE
added EASY Marketing GmbH organization & connector

### PR DESCRIPTION
**Pull request by MrParano1d**

### CODE REVIEW

**Comment by MrParano1d** [comment]
@googlebot I signed it!

**Comment by MrParano1d** [comment]
@googlebot I signed it!

**Comment by MrParano1d** [comment]
@googlebot I signed it!

File: organizations/easymarketing.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (pfrisella) [inline organizations/easymarketing.json]: ID and filenames need to be the same

so filename should be easym.json

File: sources/easymarketing.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "EASYM_STATISTIC",
```
Comment (pfrisella) [inline sources/easymarketing.json]: ID and filenames need to be the same

so filename should be easym_statistic.json

**Comment by MrParano1d** [comment]
@googlebot I fixed it.

**Comment by MrParano1d** [comment]
Come on... I didn't change anything.

**Pull request closed without merging**